crates/router/src/connector/adyen/transformers.rs

```diff
@@ -237,10 +237,9 @@ pub struct AdyenThreeDS {
 #[derive(Debug, Clone, Deserialize)]
 #[serde(untagged)]
 pub enum AdyenPaymentResponse {
-    Response(Response),
-    PresentToShopper(AdyenPtsResponse),
```

Comment:
Let's keep separate struct as per the use case and refactor all your changes in this file